### PR TITLE
🎉 New Source: Planhat

### PR DESCRIPTION
**Pull request by KimPlv**
## What
Add a new source with Planhat API 

## How
Extract Planhat models through the API

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- Connector version is set to `0.1.0`
    - `Dockerfile` has version `0.1.0`
- Documentation updated
    - `docs/integrations/source/planhat.md` including changelog with an entry for the initial version. 
    - `docs/integrations/README.md`

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-planhat/source_planhat/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    - "invoices"
+    - "issues"
+    - "licenses"
+    - "nps"
+    - "opportunities"
+    - "objectives"
+    - "sales"
+    - "tasks"
+    - "tickets"
+    - "users"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-planhat/source_planhat/manifest.yaml:248]: Are all these streams needed to be checked? Maybe let only the most common one like companies and users.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left some comments.

File: airbyte-integrations/connectors/source-planhat/source_planhat/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+documentationUrl: https://docsurl.com
```
[the remark this answers is not in the document]
Comment (KimPlv) [reply airbyte-integrations/connectors/source-planhat/source_planhat/spec.yaml]: added

File: airbyte-integrations/connectors/source-planhat/source_planhat/manifest.yaml (hunk 1 above)
Comment (KimPlv) [reply airbyte-integrations/connectors/source-planhat/source_planhat/manifest.yaml]: done

**Comment by marcosmarxm** [comment]
Hello @KimPlv and @lideke Planhat doesn't allow to create an account without talking to their sales team. I made the request and need their return. Airbyte today always need to have green integration tests to publish new and updates to the connectors. If Planhat team doesn't allow to have a sandbox account it won't be possible to access the contribution. Any update return to you here. In any case, thanks a lot to put a lot of effort to build a new connector to the project 🎖️

**Comment by marcosmarxm** [comment]
I'm in conversation with Planhat to get a sandbox account. Hope to have one created this week.

**Comment by KimPlv** [comment]
> I'm in conversation with Planhat to get a sandbox account. Hope to have one created this week.

Hello @marcosmarxm, thanks for the updates. We also have contacts with Planhat, if you need help with the process.

**Comment by marcosmarxm** [comment]
Until now Planhat didn't return Airbyte request for a sandbox account to run tests.

**Comment by marcosmarxm** [comment]
> > Until now Planhat didn't return Airbyte request for a sandbox account to run tests.
> 
> @marcosmarxm We have a sandbox where i can give you acces to validate the connector. Can you send in slack the email you want to use for the sandbox ? Thanks

Yes, sorry I missed your comment. Can you send me a Slack message?

**Comment by marcosmarxm** [comment]
Hi @KimPlv, apologies for the delay in updating and reviewing your contribution. Regrettably, we had to decline it for now because the team was unable to set up a sandbox and conduct integration tests. We understand this can be disappointing and we want to assure you that we recognize this issue. We dedicated a significant amount of effort in the previous quarter, and will continue to do so in the next to improve the process for contributions like new connectors. The team will include a new process for contributing and publishing sources in future doesn't depend on Airbyte needs to have a proper sandbox to accept contributions and ensure all tests are valid. Please get in touch if you want to learn more about it and resubmit the amazing source :) Thanks a lot for all effort you put into this contribution and feel free to contact me on Slack

**Pull request closed without merging**